### DISCUSSION TITLE
Update Plugin: Safe to do in one restart?

### DISCUSSION THREAD

**Discussion by sad-dev** [NationalSecurityAgency/ghidra#6779] [Q&A]
When installing a new version of a plugin I usually restart Ghidra twice - once to uninstall the old version and another to install the new. Is this necessary or can I do both in one go?

**Comment by dragonmacher** [comment]
I haven't run into any issues doing both steps at the same time.  You are likely to be safe, but may see issues if the plugins are making use of static data structures.
